src/test/java/net/minecraftforge/test/CriticalHitEventTest.java

```diff
@@ -22,11 +22,11 @@ public void init(FMLInitializationEvent event)
     @SubscribeEvent
     public void onCriticalHit(CriticalHitEvent  event)
     {
-        if(ENABLE)
+        if (ENABLE)
         {
             event.setResult(Result.ALLOW); //Every hit is Critical
-            //event.setResult(Result.Deny);//No hit will be Critical
-            //event.setResult(Result.Default); //Vanilla Hits
+            //event.setResult(Result.DENY);//No hit will be Critical
+            //event.setResult(Result.DEFAULT); //Vanilla Hits
```

Comment:
Randomness in tests is not a good idea.
Maybe check for a key modifier like shift or alt, and switch the result based on that.